chsdi/models/vector/vbs.py

```diff
@@ -50,3 +50,201 @@ class TERRITORIALREGIONEN(Base, Vector):
     name = Column('name', Text)
 
 register('ch.vbs.territorialregionen', TERRITORIALREGIONEN)
+
+
+class sim_facilities_a(Base, Vector):
+    __tablename__ = 'sim_fac_anhorung'
+    __table_args__ = ({'schema': 'militaer', 'autoload': False})
+    __template__ = 'templates/htmlpopup/sim_facilities.mako'
+    __bodId__ = 'ch.vbs.sachplan-infrastruktur-militaer_anhorung'
+    id = Column('stabil_id', Text, primary_key=True)
+    facname_de = Column('facname_de', Text)
+    facname_fr = Column('facname_fr', Text)
+    facname_it = Column('facname_it', Text)
+    fackind_text_de = Column('fackind_text_de', Text)
+    fackind_text_fr = Column('fackind_text_fr', Text)
+    fackind_text_it = Column('fackind_text_it', Text)
+    facstatus_text_de = Column('facstatus_text_de', Text)
+    facstatus_text_fr = Column('facstatus_text_fr', Text)
+    facstatus_text_it = Column('facstatus_text_it', Text)
+    validfrom = Column('validfrom', Text)
+    description_de = Column('description_de', Text)
+    description_fr = Column('description_fr', Text)
+    description_it = Column('description_it', Text)
+    doc_web = Column('doc_web', Text)
+    objname_de = Column('objname_de', Text)
+    objname_fr = Column('objname_fr', Text)
+    objname_it = Column('objname_it', Text)
+    bgdi_created = Column('bgdi_created', Text)
+    the_geom = GeometryColumn(Geometry(dimension=2, srid=21781))
```

Comment:
Since today because of some slq/geoalchemy updates you'll need to rebase and adapt geometry column declaration.